controllers/openstackcluster_controller.go

```diff
@@ -170,10 +170,16 @@ func (r *OpenStackClusterReconciler) reconcileCluster(logger logr.Logger, cluste
 			return reconcile.Result{}, errors.Errorf("failed to get control plane machine: %v", err)
 		}
 		if controlPlaneMachine != nil {
+			var apiPort int
```

Comment:
wouldn't this be place to default to 6443? If I read it correctly we're currently defaulting to `0` :)

Reply:
I forgot to set 6443 here...